crates/store/src/state.rs

```diff
@@ -985,6 +986,15 @@ impl State {
     pub async fn optimize_db(&self) -> Result<(), DatabaseError> {
         self.db.optimize().await
     }
+
+    /// Returns the unprocessed network notes, along with the next pagination token.
+    pub async fn get_unconsumed_network_notes(
+        &self,
+        page: PaginationToken,
+        limit: NonZero<usize>,
```

Comment:
Do we ever use `page` separately from `limit`? If not, it may make sense to combine them into one struct.